### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by carlansley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,35 @@
-# Check Digit < Insert name > Library
+# eslint-config
```
Comment (ramaghanta) [inline README.md]: jest-config

File: jest-preset.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// jest-preset.js
+
+// eslint-disable-next-line unicorn/prefer-module
+module.exports = {
+  preset: 'ts-jest',
+  collectCoverageFrom: ['<rootDir>/src/**', '!<rootDir>/src/**/*.spec.ts'],
```
Comment (ramaghanta) [inline jest-preset.js:6]: `'!<rootDir>/src/**/*.test.ts'`

File: package.json [hunk 3]
```diff
@@ -1,55 +1,47 @@
...
+    "ci:test": "jest --coverage=false",
+    "ci:coverage": "jest --coverage=true",
+    "ci:compile": "tsc --noEmit",
+    "ci:lint": "npm run lint",
+    "ci:style": "npm run prettier"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/checkdigit/library-template.git"
+    "url": "git+https://github.com/checkdigit/eslint-config.git"
```
Comment (ramaghanta) [inline package.json]: jest

File: package.json [hunk 4]
```diff
@@ -1,55 +1,47 @@
...
   },
+  "keywords": [
+    "eslint"
```
Comment (ramaghanta) [inline package.json]: jest

File: package.json [hunk 5]
```diff
@@ -1,55 +1,47 @@
...
+  ],
   "author": "Check Digit, LLC",
-  "license": "UNLICENSED",
+  "license": "MIT",
   "bugs": {
-    "url": "https://github.com/checkdigit/library-template/issues"
+    "url": "https://github.com/checkdigit/eslint-config/issues"
```
Comment (ramaghanta) [inline package.json]: +1

File: package.json [hunk 6]
```diff
@@ -1,55 +1,47 @@
...
   },
-  "homepage": "https://github.com/checkdigit/library-template#readme",
+  "homepage": "https://github.com/checkdigit/eslint-config#readme",
```
Comment (ramaghanta) [inline package.json]: +1

**Review by ramaghanta — CHANGES_REQUESTED** [review]
few minor copy-paste errors

File: jest-preset.js (hunk 2 above)
Comment (carlansley) [reply jest-preset.js:6]: I eliminated that on purpose.  wallaby doesn't seem to work with those files, and I think unused .test.ts code is less than ideal.

**Review by ramaghanta — APPROVED** [review]
lgtm

**Pull request merged**